lnwallet/interface.go

```diff
@@ -159,12 +159,14 @@ type WalletController interface {
 	// ConfirmedBalance returns the sum of all the wallet's unspent outputs
 	// that have at least confs confirmations. If confs is set to zero,
 	// then all unspent outputs, including those currently in the mempool
-	// will be included in the final sum.
+	// will be included in the final sum. The account parameter serves as a
+	// filter to retrieve the balance for a specific account. When empty,
+	// the confirmed balance of all wallet accounts is returned.
 	//
 	// NOTE: Only witness outputs should be included in the computation of
 	// the total spendable balance of the wallet. We require this as only
 	// witness inputs can be used for funding channels.
-	ConfirmedBalance(confs int32) (btcutil.Amount, error)
+	ConfirmedBalance(confs int32, accountFilter string) (btcutil.Amount, error)
```

Comment:
Non-blocking comment, but we could instead make the `accountFilter` itself into a predicate, which would allow callers to filter several accounts at once w/o needing to do repeated calls to the method. We could then have a default higher level function like `SingleAccountFilter(acctName string)` for teh more common case.

Reply:
Implementing this would also affect `ListUnspentWitness` so I've decided to leave it out for a potential follow-up instead.